rllib/connectors/learner/add_one_ts_to_episodes_and_truncate.py

```diff
@@ -101,10 +102,23 @@ def __call__(
         # batch: - - - - - - - T B0- - - - - R Bx- - - - R Bx
         # mask : t t t t t t t t f t t t t t t f t t t t t f
 
+        # TODO (sven): Same situation as in TODO below, but for multi-agent episode.
+        #  Maybe add a dedicated connector piece for this task?
+        # We extend the MultiAgentEpisode's ID by a running number here to make sure
```

Comment:
Ah tricky. This kind of trick needs to also go into the connector docs. This can solve problems, but we need to know how.

Reply:
Yup, it's getting to a point, where the default pipelines do become quite complex. We should spend some time soon to maybe simplify these again or to make the ConnectorV2 helper methods even better, e.g. `self.foreach_batch_item_change_in_place`.